### PR TITLE
User follow

### PR DESCRIPTION
**Pull request by ChanReMin**
user/artist create follow

### CODE REVIEW

File: app/routes/artist/__init__.py [hunk 1]
```diff
@@ -1,11 +1,13 @@
 from flask import Blueprint, render_template
 
 from app.decorators import login_required
-from .music import artist_music_bp
+from app.routes.artist.music import artist_music_bp
+from app.routes.artist.profile import artist_profile_bp
 from ...models.enum.account_role import AccountRole
```
Comment (ChanReMin) [inline app/routes/artist/__init__.py:6]: fix to app.models....

File: app/templates/base.html [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
 
 {% endblock %}
-
+<script src="/static/js/config.js"></script>
```
Comment (ChanReMin) [inline app/templates/base.html:21]: delete

File: app/routes/api/user/follow/__init__.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+from flask import Blueprint, jsonify, request
+from app.models.base.extended_account import ExtendedAccount
+from app.models.follow import Follow
+api_user_follow_bp = Blueprint('api_user_follow', __name__)
+@api_user_follow_bp.route('/create', methods=['POST'])
+def follow():
+    if request.method == 'POST':
```
Comment (ducchung2049) [inline app/routes/api/user/follow/__init__.py:7]: delete

File: app/config.py [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import os
 
+os.environ["DEFAULT_AVATAR_URL"] = "https://res.cloudinary.com/dddiwftri/image/upload/v1731338036/default_avatar_crdcce.svg"
```
Comment (ducchung2049) [inline app/config.py:3]: bo vao .env

**Pull request merged**